openfisca_us/variables/gov/states/il/tax/income/personal_exemption/il_personal_exemption.py

```diff
@@ -0,0 +1,50 @@
+from openfisca_us.model_api import *
+
+
+class il_personal_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IL personal exemption amount"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.il.tax.income.personal_exemption.personal_exemption
```

Comment:
rename the folder to `exemptions`

Reply:
[rename of exemption parameter structure](https://github.com/PolicyEngine/openfisca-us/pull/1056/commits/3a4a3fad34622fa880305934ee7eb451e35c2956)